apps/tsilhqotin-language-hub/src/Components/Pages/Song/Song.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card } from '@mui/material';
+
+export type SongData = {
+    id: string;
+
+    title: string;
+
+    titleEnglish?: string;
+
+    contributions: string;
+
+    lyrics: string;
+
+    audioURL: string;
+};
+
+/* eslint-disable-next-line */
+export interface SongsDetailComponentProps {
+    songData: SongData;
+}
+
+export function Song(props: SongsDetailComponentProps) {
+    const { songData } = props;
+
+    const { id, title, titleEnglish, contributions, lyrics, audioURL } = songData;
+
+    return (
+        <div>
+            <div id="heading">
+                <div id="container">
+                    <h1 id="pageTitle"> {`${title || titleEnglish}`}</h1>
+                </div>
+            </div>
+            <Card>
+                <div>English: {titleEnglish}</div>
+                <audio src={`${audioURL}`} controls />
+                <div>Contributions: {`${contributions}`}</div>
+                <div>Lyrics: {`${lyrics}`}</div>
+                <div>URL: {`${audioURL}`}</div>
```

Comment:
I wouldn't display the URL to the end user. Maybe this is for diagnostics, but remove it before you merge.